### DISCUSSION TITLE
Front page layout has been broken for a few days now

### DISCUSSION THREAD

**Discussion by alexrp** [community/community#55122] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Not sure if I'm the only one seeing this, but it broke a few days ago (125% zoom):

![image](https://github.com/community/community/assets/44076/bcc4afe7-04a9-4c26-9780-e923a73e4fc7)

I have to zoom out to 90% to get a reasonable layout:

![image](https://github.com/community/community/assets/44076/39642698-5a8f-4857-abf6-e2e891fd7f9d)

(But that's near-unusable with my screen size.)

**Comment by realrufans** [comment]
You’re not alone.

**Reply by taux1c** [reply]
Same

**Comment by Jemeni11** [comment]
It's also broken on mobile

**Reply by Jemeni11** [reply]
![Screenshot_20230514-091016](https://github.com/community/community/assets/52603291/5044cd3c-b7c6-4d02-a331-30d4c4cfccda)

**Comment by SafwenSoker** [comment]
Same here

**Comment by mmiscool** [comment]
Please fix this.  It is really annoying.

**Comment by MitofanChi** [comment]
You’re not alone.!!
